patrat.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML>
+<html LANG="RO">
+    
+<head>
+    <TITLE>Patrat Roz</TITLE>
+    <LINK HREF="STIL.CSS" REL="STYLESHEET">
```

Comment:
pentru a evita problemele ce pot aparea in linux, numele fisierelor se scriu exact cum sunt ele cu adevarat (in cazul asta, cu litere mici vad ca exista fisierul. te rog modifica